src/kvms.js

```diff
@@ -40,10 +40,17 @@ module.exports = async (config, manifest, purgeDeleted) => {
     const encrypted = kvm.encrypted === 'true' || kvm.encrypted === true
     delete kvm.encrypted
 
-    const newkvm = {
-      'name': kvmName,
-      'encrypted': encrypted,
-      'entry': Object.keys(kvm).map(key => ({ 'name': key, 'value': getValue(kvm[key]) }))
+    if (apigee.config.hybrid) {
```

Comment:
newKvm should be declared outside of if/else statement:
```
const newKvm = {
  'name': kvmName
}
if(){
newKvm.encrypted = true
} else {
etc.
}
```

Reply:
Resloved in [4a6320f](https://github.com/appythings/apigee-apim/commit/4a6320fda06bdb5eeca49ae32d871f997d35fb90)